### DISCUSSION TITLE
BluePill & Native USART bootloader

### DISCUSSION THREAD

**Discussion by ogogon** [stm32duino/Arduino_Core_STM32#1424] [Q&A]
Please tell me which download mode and programmer should I specify in the Arduino IDE, if I want to download the compiled program via USART, using the native BluePill bootloader? (If i am not mistaken,  this is AN3155...)

**Comment by fpistm** [comment]
Hi @ogogon 

I guess you are looking for: https://github.com/stm32duino/wiki/wiki/Upload-methods#serial
